mobile/lib/features/wallet/balance_change_notifier.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+import 'package:get_10101/bridge_generated/bridge_definitions.dart';
+
+class BalanceChangeNotifier extends ChangeNotifier {
+  Balance balance = Balance(onChain: 0, offChain: 0);
```

Comment:
🔧 I would avoid using API model code in a change notifier. I think this should be encapsulated in a `service`, and we should map it into a Dart class that handles the balance. That way we can also use the Dart model `Amount` for representing the balance.